### PR TITLE
Fix/CI-62/add get card and fix flow

### PR DESCRIPTION
**Pull request by julianromero-bc**
## Description 
Added get card request with logic for showing order my card if no cards created
Fix initial flow for preventing request when module not active

### CODE REVIEW

File: packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardOrder/index.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+}
+
+const CardOrder = ({ handleOpenOrderMyCard }: Props) => (
+  <IntroCardWrapper>
+    <Image name='intro-card' />
+
+    <SceneHeaderText>
+      <FormattedMessage
+        id='scenes.debit_card.intro.content.header'
+        defaultMessage='Your Gateway To The Blockchain Debit Card'
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardOrder/index.tsx:57]: should this say `Blockchain.com Debit Card`?

File: packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts [hunk 2]
```diff
@@ -7,14 +8,30 @@ import { selectors } from 'data'
 import { actions as A } from './slice'
 
 export default ({ api }: { api: APIType }) => {
+  const getCards = function* () {
+    try {
+      const data = yield call(api.getDCCreated)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:13]: why no use of a Loading remote state?  how are you able to render a loading indicator to user?

File: packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts [hunk 3]
```diff
@@ -7,14 +8,30 @@ import { selectors } from 'data'
...
+
   const getProducts = function* () {
     const debitCardModuleEnabled = (yield select(
       selectors.core.walletOptions.getWalletDebitCardEnabled
     )).getOrElse(false)
     if (debitCardModuleEnabled) {
       try {
-        const data = yield call(api.getDCProducts)
-        yield put(A.getProductsSuccess(data))
+        const products = yield call(api.getDCProducts)
```
Comment (schnogz) [inline packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:27]: same here. why no Loading remote state?

**Review by schnogz — COMMENTED** [review]
Left some comments/questions but nothing blocking the merge if you see fit

File: packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardOrder/index.tsx (hunk 1 above)
Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/scenes/DebitCard/CardOrder/index.tsx:57]: You might be mixing things. This is the text shown below the cards image. 
Here is the UX:
https://www.figma.com/file/EfbmFeqaZV7jF8XLpjGMP9/Web---Debit-Card?node-id=2%3A105

File: packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts (hunk 2 above)
Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:13]: This is requested at the beginning of the wallet page, no loading was defined to be shown anywhere

File: packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts (hunk 3 above)
Comment (julianromero-bc) [reply packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/sagas.ts:27]: same response I guess 😅 
This does not render something after request is done. I left error actions for showing some kind of error message to the user in the future

**Review by schnogz — APPROVED** [review]

**Pull request merged**